container-engine-lib/lib/backend_impls/docker/docker_manager/docker_manager.go

```diff
@@ -1660,6 +1660,14 @@ func (manager *DockerManager) getImagePlatform(ctx context.Context, imageName st
 	return imageInspect.Architecture, nil
 }
 
+func (manager *DockerManager) GetOriginalEntryPointAndCommand(ctx context.Context, imageName string) ([]string, []string, error) {
```

Comment:
as far as docker manager is concerned, we're not changing the entrypoint and command. wdyt abt `GetEntryPointAndCommand`?